kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -839,7 +845,8 @@ object KyuubiConf {
   val AUTHENTICATION_CUSTOM_CLASS: OptionalConfigEntry[String] =
     buildConf("kyuubi.authentication.custom.class")
       .doc("User-defined authentication implementation of " +
-        "org.apache.kyuubi.service.authentication.PasswdAuthenticationProvider")
+        "org.apache.kyuubi.service.authentication.PasswdAuthenticationProvider " +
+        "or org.apache.kyuubi.service.authentication.TokenAuthenticationProvider")
```

Comment:
we shall involve two new config items.

`kyuubi.authentication.custom.basic.class` and `kyuubi.authentication.custom.bearer.class`

Reply:
And for 
`kyuubi.authentication.custom.basic.class`, it should fallback to `AUTHENTICATION_CUSTOM_CLASS`

.fallbackConf(AUTHENTICATION_CUSTOM_CLASS)